packages/rocketchat-theme/client/imports/slider.css

```diff
@@ -0,0 +1,72 @@
+*,
+*::before,
+*::after {
+	box-sizing: border-box;
+}
+
+.range-slider {
+	margin: 0 0 0 0%;
+	width: 100%;
+}
+
+.range-slider__range {
+	-webkit-appearance: none;
+	width: calc(100% - (73px));
+	height: 10px;
+	border-radius: 5px;
+	outline: none;
+	padding: 0;
+	margin: 0;
+}
+
+.range-slider__range::-webkit-slider-thumb {
+	-webkit-appearance: none;
```

Comment:
we already use autoprefixer on css build, so you can remove these browser prefixes.